src/server/supplements/recorder/codeGenerator.ts

```diff
@@ -168,8 +168,11 @@ export class CodeGenerator extends EventEmitter {
     const text = [];
     if (this._options.generateHeaders)
       text.push(languageGenerator.generateHeader(this._options));
-    for (const action of this._actions)
-      text.push(languageGenerator.generateAction(action));
+    for (const action of this._actions) {
+      const actionText = languageGenerator.generateAction(action);
```

Comment:
Did we ever have new lines to split blocks of code? I expected an explicit null check instead of empty string but if they are equivalent then I don't care.

Reply:
Same thing, yes